kinode/packages/app_store/app_store/src/lib.rs

```diff
@@ -174,11 +185,11 @@ fn handle_message(
             Req::FTWorkerResult(r) => {
                 println!("app store: got weird ft_worker result: {r:?}");
             }
-            Req::Eth(e) => {
+            Req::Eth(sub) => {
                 if source.node() != our.node() || source.process != "eth:distro:sys" {
                     return Err(anyhow::anyhow!("eth sub event from weird addr: {source}"));
                 }
```

Comment:
note here: with current passthrough, this would hit wall at source != our

Reply:
If we remove this assertion, we could receive fraudulent updates from just about anyone -- if the passthrough is in the runtime, we should manipulate the `source` field to look like it's us when we get a good passthrough message.